hloc/extractors/r2d2.py

```diff
@@ -0,0 +1,55 @@
+import sys
+from pathlib import Path
+import torch
+import torchvision.transforms as tvf
+
+from ..utils.base_model import BaseModel
+
+r2d2_path = Path(__file__).parent / "../../third_party/r2d2"
+sys.path.append(str(r2d2_path))
+from extract import load_network, NonMaxSuppression, extract_multiscale
```

Comment:
This should require a `# noqa: E402` per `flake8`

Reply:
All other feature extractors do the relative import like this. Should I add this `# noqa: E402` to all the models?